### ISSUE TITLE
JS error when new formset is added

### ISSUE THREAD

**Issue by mp3il** [jazzband/django-smart-selects#210]
## Steps to reproduce

1. adding new formset (usually in a nested tabular)
2. getting js error

## Actual behavior
bindfields.js:32 Uncaught TypeError: Cannot read property 'indexOf' of undefined
    at HTMLDocument.<anonymous> (bindfields.js:32)
    at HTMLDocument.dispatch (jquery.js:4737)
    at HTMLDocument.elemData.handle (jquery.js:4549)
    at Object.trigger (jquery.js:7819)
    at HTMLDocument.<anonymous> (jquery.js:7903)
    at Function.each (jquery.js:365)
    at jQuery.fn.init.each (jquery.js:137)
    at jQuery.fn.init.trigger (jquery.js:7902)
    at t.value (jquery.djangoformset.js:284)
    at HTMLAnchorElement.<anonymous> (jquery.djangoformset.js:78)
## Expected behavior

No js error ;)

my guess is that the following changes at lines 31-33:
```
        var chainfield = $(chained).attr("data-chainfield");
        **if (!chainfield)
            return;**
```
can fix it, but would like to confirm with you guys :)

**Comment by blag** [comment]
@mp3il Can you test the `master` branch to see if it's fixed for you now that I merged in #225?

**Comment by manelclos** [comment]
Closing due to inactivity.
